StellarDotnetSdk/Operations/InvokeHostFunctionOperation.cs

```diff
@@ -108,22 +108,25 @@ public class CreateContractOperation : InvokeHostFunctionOperation
     /// <summary>
     ///     Creates a new <c>CreateContractOperation</c> using the provided address and salt.
     /// </summary>
-    /// <param name="wasmId"></param>
+    /// <param name="wasmHash">
+    ///     (aka Wasm ID or hash ID or just hash), a hex-encoded string of the unique identifier of a
+    ///     WebAssembly (Wasm) bytecode that represents a compiled smart contract on the Stellar network.
+    /// </param>
     /// <param name="accountId"></param>
-    /// <param name="salt"></param>
+    /// <param name="salt">(Optional) Custom salt 32-byte salt for the token ID. It will be randomly generated if omitted.</param>
     /// <param name="sourceAccount">(Optional) Source account of the operation.</param>
-    public static CreateContractOperation FromAddress(string wasmId, string accountId, byte[]? salt = null,
```

Comment:
Do you mean we need to add a note for this specific method? Because there is already a more general breaking change note about that: "Renamed wasmId to wasmHash to be more consistent with the official documentation and other SDKs".

Reply:
Ahh yeah just wanted to put in a reminder taht we don't forget it in the release notes.